### ISSUE TITLE
Make it faster to add filament changes at layer so you don't have to slice between every command

### ISSUE THREAD

**Issue by Cetacea37** [bambulab/BambuStudio#1977]
**Is your feature request related to a problem? Please describe.**
After you slice the file, you have the slider on the preview page and can slide it to a layer and push the + sign and do things like have it change filaments at that layer.  In Prusaslicer, you could add as many of these as you wanted, then hit Slice.  In Bambu, as soon as you add one the slider bar vanishes and you have to slice the file again before you can add another.  With the introduction of HueForge, more people are going to be using this feature, and it's very time consuming to have to wait for it to slice again after every single color change you want it to do.  

**Describe the solution you'd like**
It would be much better if it worked like Prusaslicer which allows you to add in all the layer change commands you want, then you just slice them all at once.  I don't want the slider to disappear when I add a filament change.  I want to be able to go through and add all my layer change commands and then just hit Slice when I'm done.

**Describe alternatives you've considered**
Far as I know the only option is to do each change and reslice before you can do the next one.

**Additional context**

**Comment by nordurljosahvida** [comment]
Yes, please let's consider this. No need to reslice after adding a color change as that does not backwards affect any layer information. Thanks
